libs/utils/src/http/json.rs

```diff
@@ -27,6 +27,26 @@ pub async fn json_request<T: for<'de> Deserialize<'de>>(
         .map_err(ApiError::BadRequest)
 }
 
+pub async fn json_request_maybe<T: for<'de> Deserialize<'de> + Default>(
```

Comment:
Could you add to the rustdocs of this function that it should only be used in test endpoints? cplane's code generation tool can't deal with smart schemas for http requests, so it's better to save people from learning that the hard way.

Reply:
https://github.com/neondatabase/neon/pull/9723/commits/681fb1bbf78f967878ab8653d89cfb6b88e797c3